packages/cardano-services/test/StakePoolSeach/DbSyncStakePoolSearchProvider/StakePoolSearchBuilder.test.ts

```diff
@@ -27,18 +30,27 @@ describe('StakePoolSearchBuilder', () => {
   describe('queryRetirements', () => {
     it('queryRetirements', async () => {
       const retirements = await builder.queryRetirements([1, 2, 3]);
-      expect(retirements).toMatchSnapshot();
+      const striptedIds = retirements.map((r) => {
+        const { hashId, ...rest } = r;
+        return rest;
+      });
+      expect(striptedIds).toMatchSnapshot();
     });
   });
   describe('queryRegistrations', () => {
     it('queryRegistrations', async () => {
       const registrations = await builder.queryRegistrations([1, 2, 3]);
-      expect(registrations).toMatchSnapshot();
+      const striptedIds = registrations.map((r) => {
+        const { hashId, ...rest } = r;
+        return rest;
+      });
+      expect(striptedIds).toMatchSnapshot();
```

Comment:
We're not expecting the stripped IDs to match the snapshot :smile: Always think of the person reading this for the first time :+1: 
```suggestion
      expect(registrations).toMatchSnapshot();
```